federated_learning/openfl/openfl_mednist_2d_registration/envoy/mednist_dataset_shard_descriptor.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (C) 2020-2021 Intel Corporation
```

Comment:
thanks, I have an impression that the licensing blob should follow the licensing informantion here https://github.com/Project-MONAI/MONAI/blame/dev/CONTRIBUTING.md#L66-L77, could you please revise?

Reply:
@wyli, thanks for comment, we think it is better to remove the license information since this is an example of user code by purpose